tests/feature/UniquePinnedPostTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * WP Curate Tests: Unique Pinned Post Test
+ *
+ * @package wp-curate
+ */
+
+namespace Alley\WP\WP_Curate\Tests\Feature;
+
+use Alley\WP\WP_Curate\Tests\Test_Case;
+
+/**
+ * A test suite for unique pinned posts.
+ *
+ * @link https://mantle.alley.com/testing/test-framework.html
+ */
+class UniquePinnedPostTest extends Test_Case {
+	/**
+	 * Run before each test.
+	 */
+	protected function setUp(): void {
+		parent::setUp();
+
+		$this->set_permalink_structure( '/%postname%/' );
+
+		static::factory()->post->create(
+			[
+				'post_title' => 'TheWrapTestPost 1',
```

Comment:
We shouldn't have client names in tests in open source repos. Let's just make this "Test Post 1" etc for others in this file.